here-naksha-lib-psql/src/main/java/com/here/naksha/lib/psql/PsqlInstanceConfig.java

```diff
@@ -40,6 +40,16 @@
 @SuppressWarnings("unused")
 public class PsqlInstanceConfig {
 
+  public static final String HOST = "host";
+
+  public static final String PORT = "port";
+
+  public static final String DB = "db";
+  public static final String USER = "user";
+  public static final String PASSWORD = "password";
```

Comment:
extracted these to constants so I can reference them from `StorageApiTask` instead of basing logic on hardcoded value